homeassistant/components/esphome/climate.py

```diff
@@ -232,12 +268,21 @@ def hvac_action(self) -> Optional[str]:
     @esphome_state_property
     def fan_mode(self):
         """Return current fan setting."""
-        return _fan_modes.from_esphome(self._state.fan_mode)
+        if self._state.custom_fan_mode:
+            return self._state.custom_fan_mode
```

Comment:
You can change the `elif:` to `if:` to fix this error. `else: return` can be replaced by a basic `return`. So:
```
if:
  return
if:
  return
return
```

Reply:
Fixed, thanks